app/views/library/LibraryMainView.vue

```diff
@@ -0,0 +1,322 @@
+        <h2 class="library__heading-text pixel-text">
+          Give your members access to the most engaging way to learn coding
+        </h2>
+        <button class="btn library__heading-btn btn-moon" @click="toggleLicenseModal">
+          Contact Us
+        </button>
+      </section>
+
+      <section class="library__sub">
+
+        <div class="row library__sub-1">
+          <div class="library__sub-text col-md-6">
+            Libraries play an important role in advancing educational equity in their communities. Many young learners do not have access or exposure to computer science, and libraries can provide important resources to these underserved groups. That is why we are partnering with libraries across the country to provide access to our game-based learning tools and educational resources.
+          </div>
+          <div class="library__sub-image col-md-6">
+            <img src="/images/pages/library/library-students.png" alt="Students in library" class="library__sub-image-img img-responsive">
+          </div>
+        </div>
+
+        <div class="row library__sub-2">
+          <div class="library__sub-image col-md-6">
+            <img src="/images/pages/library/teach-real-code.png" alt="Teach real code" class="library__sub-image-img img-responsive">
+          </div>
+          <div class="library__sub-text col-md-6">
+            Our games teach real typed Python, JavaScript and C++ coding languages through the power of play. The gameplay is paired with a scaffolded curriculum to allow self-paced learners the ability to master foundational programming concepts before moving on to the more complex. Each level challenges the player to type the code necessary to advance their hero and achieve the level goal.
+          </div>
+        </div>
+
+      </section>
+
+      <section class="library__needs">
+        <div class="library__needs__heading pixel-text">
+          Our platform is built to address the specific needs of libraries and their patrons:
+        </div>
+        <div class="row library__needs__info">
+          <div class="col-md-6 library__needs__info__text">
+            <div class="library__needs__info__text-heading">
+              Accessible
+            </div>
+            <div class="library__needs__info__text-sub">
+              A basic computer with internet access is all you need to sign-in and start playing. The content is intended for beginners with no prior coding experience.
+            </div>
+          </div>
+          <div class="col-md-6 library__needs__info-img">
+            <img src="/images/pages/library/kids-in-library.jpeg" alt="Students with computer" class="img-responsive">
+          </div>
+        </div>
+
+        <div class="row library__needs__info">
+          <div class="col-md-6 library__needs__info-img">
+            <img src="/images/pages/library/self-paced-intuitive.png" alt="Self paced and intuitive" class="img-responsive">
```

Comment:
This image file is pretty big. Can we zoom in on it, resize it, switch file format, or compress it differently? It's also hard to read anything because of the contrast between the display resolution and the file resolution. Cropping in order to zooming into the error message itself, perhaps, might help.